css/style.css

```
@@ -1,33 +1,18 @@
 @import "reset.css";
+@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css");
 
-/* components */
-@import "components/nav.css";
-@import "components/banner.css";
-@import "components/articles.css";
-@import "components/footer.css";
-
-@font-face {
-  font-family: "Pretendard-Regular";
-  src: url("https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff")
-    format("woff");
-  font-weight: 400;
-  font-style: normal;
-}
+/* pages */
+@import "pages/main/components/nav.css";
+@import "pages/main/components/banner.css";
+@import "pages/main/components/articles.css";
+@import "pages/main/components/footer.css";
+@import "pages/login/login.css";
 
 * {
   box-sizing: border-box;
 }
 
 body {
-  font-family: "Pretendard-Regular";
-  width: 100vw;
-}
-
-button {
-  border: none;
-}
-
-a {
-  text-decoration: none;
-  color: inherit;
+  font-family: "Pretendard Variable";
+  height: 100vh;
```

❗️ 수정요청
body에 height가 100vh 로 제한되는 것이 이상한 것 같아요~ min-height 값을 의도하셨던 것이 아닌가합니다.

```suggestion
  min-height: 100vh;
```